app/views/candidate_interface/start_page/create_account_or_sign_in.html.erb

```diff
@@ -4,31 +4,39 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= form_with(
-      model: @create_account_or_sign_in_form,
-      url: candidate_interface_create_account_or_sign_in_path(providerCode: params[:providerCode], courseCode: params[:courseCode]),
-      method: :post,
-    ) do |f| %>
-      <%= f.govuk_error_summary %>
+    <h1 class="govuk-heading-xl">
+      <%= t('page_titles.create_account_or_sign_in') %>
+    </h1>
 
-      <h1 class="govuk-heading-xl">
-        <%= t('page_titles.create_account_or_sign_in') %>
-      </h1>
+    <% if FeatureFlag.active?(:one_login_candidate_sign_in) %>
+      <p class="govuk-body">
+        <%= t('govuk.one_login_account_guidance') %>
+      </p>
 
-      <%= f.govuk_radio_buttons_fieldset :existing_account, legend: { text: 'Do you already have an account?' } do %>
-        <%= f.govuk_radio_button :existing_account, true, label: { text: 'Yes, sign in' }, link_errors: true do %>
-          <%= f.govuk_email_field :email, label: { text: 'Email address', size: 's' }, hint: { text: 'Enter the email address you used to register, and we will send you a link to sign in.' }, width: 'two-thirds', autocomplete: 'email', spellcheck: false %>
+      <%= govuk_button_to(t('continue'), OneLogin.bypass? ? '/auth/one-login-developer' : '/auth/onelogin') %>
```

Comment:
```suggestion
      <%= govuk_button_to(t('continue'), OneLogin.bypass? ? '/auth/one-login-developer' : '/auth/one-login') %>
```

Reply:
This needs to change to `one_login` because it's based on the name of the provider, that's why I didn't change it